exercises/affine-cipher/affine_cipher_test.rb

```diff
@@ -1,54 +1,69 @@
 require 'minitest/autorun'
 require_relative 'affine_cipher'
 
-class AffineTest < Minitest::Test
+# Common test data version: 1.0.0 377d917
+class AffineCipherTest < Minitest::Test
   def test_encode_yes
-    #skip
+    # skip
     cipher = Affine.new
     cipher.addkey(5, 7)
```

Comment:
Do we need a separate addkey method?
Why can we not just do: `Affine.new(5,7)`

Reply:
That's a good point. I think my reasons were twofold:
1. While developing this exercise I first made programs in bash and ruby with no tests.  I have since developed the Affine class from that ruby program into one that can be instanciated, load text, and have it's keys change. Essentially I set it up so the object could be used to brute force against a ciphertext with a lost key easily.
  - I realize that this implementation necessitated by the tests.
2. I thought it would be clearer that the keys were being added if they had their own function.
  - Although initially I wanted the keys to be added with `cipher.key = [5, 7]` or something.
#### I realize it will probably be more idiomatic and fitting for exercism if the keys are just added with new.
I'll make that change in the generator if you think it is best.